docs/reference/upgrade.asciidoc

```diff
@@ -9,14 +9,20 @@ process so upgrading does not interrupt service. Rolling upgrades are supported:
 * Between minor versions
 * From 5.6 to 6.8
 * From 6.8 to {version}
+* Between {minor-version} versions
 
 
 The following table shows the recommended upgrade paths to {version}.
 
-[cols="<1m,3",options="header",]
+[cols="<1,3",options="header",]
 |====
 |Upgrade from   
-|Recommended upgrade path  to {version}
+|Recommended upgrade path to {version}
+
+ifeval::[ "{version}" != "{minor-version}.0" ]
+|A previous {minor-version} version (e.g., {minor-version}.0)
```

Comment:
This seems a bit awkward since this column otherwise contains only minor version number ranges. Is there a macro for the previous patch version? If so, could we say `{minor-version}.0` if the current version is `{minor-version}.1` and `{minor-version}.0-{previous-patch-version}` otherwise?

Reply:
I agree that the wording is a bit awkward. A range would be preferable.

Unfortunately, there is no attribute/macro for the previous patch version. We could add a `{prev-patch-version}` attribute to the appropriate [Stack version attribute list](https://github.com/elastic/docs/blob/master/shared/versions/stack/7.6.asciidoc), but we'd need to include it in for every version to avoid breaks. That would include n.x.0 versions, which could create issues.

If you'd prefer a range here, it would probably be simpler for us to hard code the maintenance releases and add an item to our release prep process.

Reply:
Ok, let's leave it like this.